gallery/workbook/Azure Monitor.json

```diff
@@ -181,8 +181,7 @@
 				{
 					"id": "Workbooks/Azure Monitor - Agents/Agent Migration Tracker",
 					"author": "Microsoft",
-					"name": "AMA Migration Helper",
-					"isPreview": true
+					"name": "AMA Migration Helper"
```

Comment:
FYI there's also description field you can set in here to have that show up in the gallery as well